benchmarks/Makefile

```diff
@@ -36,3 +36,29 @@ read-update-simulation:
 create-commits-simulation:
 	./gradlew gatlingRun --simulation org.apache.polaris.benchmarks.simulations.CreateCommits \
       -Dconfig.file=./application.conf
+
+.PHONY: reports-list
+reports-list:
+	@for report in $$(ls -d build/reports/gatling/*/ 2>/dev/null | sort -r); do \
+		basename="$$(basename $$report)"; \
+		name="$$(echo $$basename | sed 's/-[0-9]\{17\}$$//')"; \
+		timestamp="$$(echo $$basename | grep -o '[0-9]\{17\}$$')"; \
+		if [ -n "$$timestamp" ]; then \
```

Comment:
nit: is it actually possible for a Gatling report to *not* have any timestamp... ?